docs/parameters.rst

```diff
@@ -1,277 +1,179 @@
 .. _standard-parameters:
 
-Parameters
-==========
+Standard parameters
+===================
 
-OpenSCM follows a number of conventions related to parameters, we document these
-In OpenSCM a 'parameter' is anything which is passed to a model e.g. |CO2| emissions, equilibrium climate sensitivity, aerosol forcing scaling.
+In OpenSCM a ':ref:`parameter <parameter-hierarchy>`' is any named input or output variable of a mode, e.g. |CO2| emissions, equilibrium climate sensitivity, aerosol forcing scaling.
+As described :ref:`here <parameter-hierarchy>`, parameters are given in a hierarchy, e.g. ``Emissions`` -> ``CO2`` -> ``Industrial``.
 
 Simple climate models come in many different shapes and forms hence we do not expect them to all be able to do everything.
-However, to be included in OpenSCM they must be able to understand all of OpenSCM's key parameters where here, 'understand' means that their adapters should either be able to use them or should throw sensible warning/error messages if they are used.
+However, to be included in OpenSCM they should make sure their parameters fit into these standard parameters as far as possible to ensure models can be interchanged easily.
+Of course, also model-specific parameters are possible (see also :ref:`writing-adapters`).
+
 
 Conventions
 -----------
 
-'Pre-industrial' refers to an unperturbed state of the climate.
+In the following, 'pre-industrial' refers to an unperturbed state of the climate.
 Individual adapters can translate this into whatever year they need to for their model, but they should do such translations with this definition in mind.
 
 'Reference period' refers to the period a given variable is reported relative to the mean of.
 For example, 'surface temperature relative to a 1961-1990 reference period' refers to surface temperatures relative to the mean of the period 1961-1990.
 Adapters can report variables without reference periods if these are not used internally by the models.
 However, if the model uses an internal reference period then this should be indicated in the reported variable names by appending ``(rel. to XXXX-YYYY)`` to the variable name.
-Hence, 'surface temperature relative to a 1961-1990 reference period' would become ``Surface Temperature (rel. to 1961-1990)`` [TODO: discuss with other authors how we want this to play out].
+Hence, 'surface temperature relative to a 1961-1990 reference period' would become ``Surface Temperature (rel. to 1961-1990)``.
+[TODO: discuss with other authors how we want this to play out]
 
 
-Variables
----------
+Aggregation
+-----------
 
-Variables in OpenSCM come as part of a hierarchy, separated by the ``|`` character.
+Parameters in OpenSCM come as part of a hierarchy, in the following separated by the ``|`` character.
```

Comment:
```suggestion
Parameters in OpenSCM come as part of a hierarchy, separated by the ``|`` character.
```

They're always separated by this character no? Or do you envisage being able to separate different hierarchical separators?

Reply:
Not in the API...

Reply:
definitely outdated/unthinking comment by me, oops